src/sections/Home/Partners-home/partnerSection.style.js

```diff
@@ -1,103 +1,94 @@
 import styled from "styled-components";
 const PartnerItemWrapper = styled.section`
-    padding: 2rem 0;
-    margin: 0rem 0rem 5rem 0rem;
-    overflow: hidden;
-    .section-title{
-        h4{
-            color: ${props => props.theme.grey737373ToGrey4C4A4A};
-            text-align: center;
-            margin-top: .5rem;
-            transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-        }
-    }
-    .partner-image{
-        margin-left:1.5rem;
-        margin-right:1.5rem;
+  padding: 2rem 0;
+  margin: 0rem 0rem 5rem 0rem;
+  overflow: hidden;
+  .section-title {
+    h4 {
+      color: ${(props) => props.theme.grey737373ToGrey4C4A4A};
+      text-align: center;
+      margin-top: 0.5rem;
+      transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
     }
-    .partner-slider{
-        display: flex;
-        justify-content: center;
-        flex-wrap: nowrap;
-        @media(max-width: 1400px){
-            flex-wrap: wrap;
-        }
+  }
+  .partner-image {
+    margin-left: 1.5rem;
+    margin-right: 1.5rem;
+  }
+  .partner-slider {
+    display: flex;
+    justify-content: center;
+    flex-wrap: nowrap;
+    @media (max-width: 1400px) {
+      flex-wrap: wrap;
```

Comment:
Can we add `screen` as well? 
Updated code can be: `@media screen and (max-width: 1400px)` ?

Reply:
sure I will make changes .